### DISCUSSION TITLE
the inability to massively put tags inside the album

### DISCUSSION THREAD

**Discussion by ilyaChuk** [immich-app/immich#12168] [Feature Request]
### The bug

![image](https://github.com/user-attachments/assets/ff04e026-834c-4bd7-ae9b-5830aabe4669)
In the normal asset view mode, there is an option to set tags in the three points menu. but when you are viewing an album, there is no such function.

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

1.113.0

### Version of Immich Mobile App

1.113.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

1. go to page `albums/someAlbum`
2. select some photos
3. open 3 dots menu
4. see the absence of this option

**Comment by mmomjian** [comment]
Thank you. This is a feature request so I have moved it.

**Comment by MaroIshiku** [comment]
I too suggest adding a bulk tagging feature to Immich, allowing users to tag multiple photos and videos at once. While the recent introduction of tags is a great improvement, the current process of tagging items individually can be time-consuming, especially for users like myself who previously relied on albums for organization. This enhancement would streamline the transition from album-based organization to a more flexible tag-based system for users with several hundreds of uploaded photos and videos.

**Comment by schizovivek** [comment]
phew ! glad I searched for this. I was doubting myself and wondering why I wasn't able to do it. Just kinda assumed there'd be multi-select tagging option implemented. Glad this has been raised as a feature request. Really looking forward for this to be implemented.

In the meanwhile is there a cli way to do this?
On searching I got this link https://immich.app/docs/api/create-tag but am unsure how to use it

**Comment by Alextoch** [comment]
Indeed, it would be great if that existing feature (tags multiple pictures) could be available on all pictures views : 
- after a search
- from a person page
- ...

**Comment by Metriximor** [comment]
I would also add that when I'm inside a tag view, if I want to add sub tags to images, I have to do it one by one, I can't select multiple and add the tag to all of them.

**Comment by bo0tzz** [comment]
I'm not sure why this is still open, it was fixed pretty much immediately in #12172
